package.json

```diff
@@ -8,7 +8,9 @@
     "test:unit": "jest --testPathPattern=Generator/tests/unit",
     "test:integration": "jest --testPathPattern=Generator/tests/integration",
     "update-docs": "npx typedoc Generator/*.ts --name GeneratorDocs",
-    "start": "tsc Generator/Demo/script.ts && ts-node Generator/Demo/app.ts"
+    "start": "tsc Generator/Demo/script.ts && ts-node Generator/Demo/app.ts",
+    "generate": "npx ts-node ./Generator/Generate.ts $npm_config_filepath $npm_config_start $npm_config_end $npm_config_batchSize",
```

Comment:
Using cross-var library will make generate command compatible with windows, linux, and mac
https://npmmirror.com/package/cross-var#:~:text=The%20goal%20of%20cross-var%20is%20to%20let%20you,how%20to%20use%20cross-var%20in%20your%20scripts.%20Usage